cmd/skywire-cli/commands/vpn/vvpn.go

```diff
@@ -0,0 +1,191 @@
+package clivpn
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/toqueteos/webbrowser"
+
+	clirpc "github.com/skycoin/skywire/cmd/skywire-cli/commands/rpc"
+	clivisor "github.com/skycoin/skywire/cmd/skywire-cli/commands/visor"
+	"github.com/skycoin/skywire/cmd/skywire-cli/internal"
+	"github.com/skycoin/skywire/pkg/servicedisc"
+	skyenv "github.com/skycoin/skywire/pkg/skyenv"
+	"github.com/skycoin/skywire/pkg/transport/network"
+	"github.com/skycoin/skywire/pkg/visor"
+	"github.com/skycoin/skywire/pkg/visor/visorconfig"
+)
+
+var timeout time.Duration
+
+func init() {
+	RootCmd.AddCommand(
+		vpnListCmd,
+		vpnUICmd,
+		vpnURLCmd,
+		vpnStartCmd,
+		vpnStopCmd,
+	)
+	vpnListCmd.Flags().StringVarP(&ver, "ver", "v", "1.0.1", "filter results by version")
```

Comment:
Should we get the default value for the version from the build tag instead maybe?